main.py

```diff
@@ -298,27 +302,33 @@ def generate_query(selector, ip_translator, mlab_site_resolver):
   start_time_datetime = selector.start_time
   end_time_datetime = start_time_datetime + datetime.timedelta(seconds = selector.duration)
 
-  network_lookup_found_blocks = ip_translator.find_ip_blocks(
-      selector.client_provider)
-  if len(network_lookup_found_blocks) == 0:
-    raise NoClientNetworkBlocksFound(selector.client_provider)
-
-  server_ips = []
-  try:
-    for retrieved_site_ip in mlab_site_resolver.get_site_ips(selector.site,
-                                                             mlab_project = selector.mlab_project):
-      server_ips.append(retrieved_site_ip)
-      logger.debug("Found IP for {site} of {site_ip} on test {test}.".format(
-          site=selector.site, site_ip = retrieved_site_ip, test = selector.mlab_project))
-  except Exception as caught_error:
-    raise MLabServerResolutionFailed(caught_error)
-
+  if selector.client_provider != None:
+    client_ips = ip_translator.find_ip_blocks(selector.client_provider)
```

Comment:
I think the `_blocks` suffix is useful because it indicates to the reader that these are not going to be like `['1.2.3.4', '2.3.4.5']` and instead they're `[(12345, 12348), ...]`. It's confusing if we name it `client_ips` and `server_ips` when they store very different data.